SPECS/dracut/dracut.signatures.json

```diff
@@ -5,6 +5,6 @@
   "megaraid.conf": "914824cdbe0c525b71efa05a75e453335b0068beb8bc28bef2a5866d74bf7dd4",
   "mkinitrd": "32a0e19de954a356a0f1903a1f75d3266493b55d71b0eeab6bd07b585e955dcf",
   "module-setup.sh": "330af5c105793fb37434730ce0ff59467a9cc60a81a5e32193dc53235e9744c1",
-  "overlayfs-mount.sh": "63552c4115b5c37ed04c5311367301483e2dfe1091dc5717b2bd22d9cdda044b"
+  "overlayfs-mount.sh": "e0d009b765fea58319f9b4b33f5399d6fea90f3ba98a4fdad43d80256e555027"
```

Comment:
Please update the spec as well - you must increment the `Release` tag and add a changelog entry.

Reply:
Fixed, please review.